src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -118,13 +119,78 @@
 (def default-allow-auth-extensions
   false)
 
+(def default-cadir
+  "/etc/puppetlabs/puppetserver/ca")
+
+(defn default-cacert
+  [cadir]
+  (str cadir "/ca_crt.pem"))
+
+(defn default-cacrl
+  [cadir]
+  (str cadir "/ca_crl.pem"))
+
+(defn default-cakey
+  [cadir]
+  (str cadir "/ca_key.pem"))
+
+;; TODO how do we get certname here?
+(def default-caname
+  "Puppet CA: $certname")
+
+;; TODO where should this actually live?
+(defn default-autosign
+  [cadir]
+  (str cadir "/autosign.conf"))
+
+(def default-allow-duplicate-certs
+  false)
+
+;; 15 years in seconds
+(def default-ca-ttl
+  473000000)
```

Comment:
We currently use five years in Puppet, but there was a request from Nick Walker approved by Eric Sorenson to make it longer.